cart/frontend/build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+plugins {
+    id("org.siouan.frontend") version "1.2.1"
+    id("java")
+}
+
+group = "com.kotlin-spring-vue"
+version = "0.0.1-SNAPSHOT"
+
+java {
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+buildscript {
+    repositories {
+        mavenCentral()
+        maven {
+            url = uri("https://plugins.gradle.org/m2/")
+        }
+    }
+}
+
+frontend {
+    nodeVersion.set("10.16.0")
```

Comment:
So you've added gradle to frontend specifically to simply run npm scripts? Why not run npm from main gradle script?

Reply:
Why is it bad? Do you propose to take everything to a higher level? (I think there will be confusion with the paths) Or do you suggest making a sh script?

This is an example from plugin devs: 
https://github.com/siouan/frontend-gradle-plugin/tree/master/examples/multi-projects-war-application

Reply:
Yes, this might work. Also, look at the existing modules in this project - one gradle per module. It's up to you how to implement this. Make sure there are clear instructions how to build, package and deploy this